examples/crowdfunding/scripts/createApp.js

```diff
@@ -0,0 +1,103 @@
+const { executeTransaction, TransactionType, SignType, update } = require('algob');
+const { decodeAddress } = require('algosdk');
+
+/**
+* Description: Converts Integer into Bytes Array
+*/
+function getInt64Bytes (x) {
+  const y = Math.floor(x / 2 ** 32);
+  const byt = [y, (y << 8), (y << 16), (y << 24), x, (x << 8), (x << 16), (x << 24)].map(z => z >>> 24);
+  return new Uint8Array(byt);
+}
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const creatorAccount = deployer.accountsByName.get('alice');
+  const donorAccount = deployer.accountsByName.get('john');
+
+  const algoTxnParams = {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: creatorAccount.addr,
+    amountMicroAlgos: 200000000,
+    payFlags: {}
+  };
+  await executeTransaction(deployer, algoTxnParams);
+
+  // Get begin date timestamp to pass in
+  const beginDateTimestamp = new Date();
+  beginDateTimestamp.setSeconds(beginDateTimestamp.getSeconds() + 2);
+
+  // Get end date timestamp to pass in
+  const endDateTimestamp = new Date();
+  endDateTimestamp.setSeconds(endDateTimestamp.getSeconds() + 12000);
+
+  // Get fund close date timestamp to pass in
+  const fundCloseDateTimestamp = new Date();
+  fundCloseDateTimestamp.setSeconds(fundCloseDateTimestamp.getSeconds() + 120000);
+
+  // convert address to bytes
+  let addr = decodeAddress(creatorAccount.addr);
+
+  // initialize app arguments
+  let appArgs = [
+    getInt64Bytes(beginDateTimestamp.getTime()),
+    getInt64Bytes(endDateTimestamp.getTime()),
+    getInt64Bytes(7000000),
+    addr.publicKey,
+    getInt64Bytes(fundCloseDateTimestamp.getTime())
+  ];
+
+  // Create Application
+  // Note: An Account can have maximum of 10 Applications.
+  const res = await deployer.deploySSC(
+    'crowdFund.teal', // approval program
+    'crowdFundClose.teal', // clear program
```

Comment:
Can we rename to `crowdFundApproval.teal` and `crowdFundClear.teal` ?

Reply:
yes